### PR TITLE
Todo app

### PR DESCRIPTION
**Pull request by taiwo2**
The look of the User Interface and it's able to do the following:

- view all projects
- view all todos in each project (probably just the title and duedate.. perhaps changing color for different priorities)
- expand a single todo to see/edit its details
- delete a todo

### CODE REVIEW

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Status: Invalid ❌ ❌ 

**Hello @taiwo2** 

### this pull request is marked as invalid for the following reasons
---

- [ ] please fix all the linters issues before requesting another code review

---

please make these changes and request another code review

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,465 @@
...
+import deleteTask from './todo/deleteTask';
+import viewTask from './todo/viewTask';
+import completedTask from './todo/completedTask';
+import storage from './storage';
+import removeProject from './project/removeProject';
+import filterTask from './project/filterTask';
+import editTask from './todo/edit';
+import editProject from './project/editProject';
+
+const format = require('date-fns/format');
```
Comment (christianotieno) [inline src/index.js]: ##  Fix the import errors with most of the files
- [ ] We have an issue importing these files on our `index.js` module. Do you mind fixing this?
```suggestion
import domListeners from './modules/domListeners';

import deleteTask from './modules/todo/deleteTask.js';
import viewTask from './modules/todo/viewTask.js';
import completedTask from './modules/todo/completedTask.js';
import storage from './modules/storage.js';
import removeProject from './modules/project/removeProject';
import filterTask from './modules/project/filterTask.js';
import editTask from './modules/todo/edit.js';
import editProject from './modules/project/editProject.js';

const format = require('date-fns/format');```

File: src/modules/project/createProject.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import displayController from '../displayController';
```
Comment (christianotieno) [inline src/modules/project/createProject.js:1]: ## Fix the `createProject` functionality 
 - [ ] A user can not create a project as is required of this project. we are currently getting an error at the console.

File: src/modules/todo/addTask.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
+import task from './todo';
```
Comment (christianotieno) [inline src/modules/todo/addTask.js:1]: ## Fix the `addTodo` task 
 - [ ] A user can not add the task as is required of this project. we are currently getting an error at the console.

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes required ♻️ 
Hello @taiwo2 👋🏾 
Congratulations on the hard work so far. We are almost there. Before we proceed we need to take care of some issues:

## Mandatory
- [ ] Fix the create project functionality
- [ ] Fix the add ToDo functionality
- [ ] Fix the import errors with most of the files 
- [ ] Fix the informing console errors.
- [ ] Move the node modules to the .gitignore file

### Happy Coding  💻 
#### Christian Otieno (CR)

**Pull request closed without merging**